src/metabase/models/table_privileges.clj

```diff
@@ -0,0 +1,17 @@
+(ns metabase.models.table-privileges
+  "Each row in the `table_privileges` table contains the privileges that the current user or group has on a given table.
+
+   The `table_privileges` table just a cache of the data returned from driver/table-privileges, but it's stored in the
+   database so that we can query it more easily."
+  (:require
+   [methodical.core :as methodical]
+   [toucan2.core :as t2]))
+
+(def TablePrivileges
```

Comment:
we don't need this anymore. we only create this for old models because old tests still referenced them.

for new models we can just use `:model/TablePrivileges`

Reply:
Cool. Fixed 147f5cef669c8f8d13cfdb4b93750289085453f7